records_mover/records/job/schema.py

```diff
@@ -17,7 +17,19 @@ def method_to_json_schema(method: Callable[..., Any]) -> JsonSchema:
     special_handling: Dict[str, List[JsonParameter]] = {
```

Comment:
This is where we do the original translation of parameters in the factory methods to the intermediate JSON Schema format.